spacy/language.py

```diff
@@ -1295,27 +1296,27 @@ def evaluate(
             kwargs = dict(scorer_cfg)
             kwargs.setdefault("nlp", self)
             scorer = Scorer(**kwargs)
-        texts = [eg.reference.text for eg in examples]
-        docs = [eg.predicted for eg in examples]
+        # reset annotation in predicted docs and time tokenization
         start_time = timer()
-        # tokenize the texts only for timing purposes
         if not hasattr(self.tokenizer, "pipe"):
-            _ = [self.tokenizer(text) for text in texts]  # noqa: F841
+            for eg in examples:
+                eg.predicted = self.tokenizer(eg.reference.text)
```

Comment:
Should this be `self.make_doc`?

Reply:
It should be identical, unless you're worried about the timing for the `max_length` check?

Reply:
Was just thinking how it might work in subclasses etc, in case they have extra logic in`make_doc`

Reply:
Yeah, but then it's also not parallel to `tokenizer.pipe` below. I would strongly hope it wouldn't make a difference, but don't mind changing it.

Reply:
Hmm, maybe since `pipe` doesn't use `tokenizer.pipe`, we should switch this 100% to `make_doc`.

Reply:
Yes let's use `make_doc`